infra/modules/setup/aws_auth.tf

```diff
@@ -0,0 +1,10 @@
+resource "kubernetes_config_map" "aws_auth" {
+  metadata {
+    name      = "aws-auth"
+    namespace = "kube-system"
+  }
+
+  data {
+    mapRoles = "- rolearn: ${var.node_role_arn}\n  username: system:node:{{EC2PrivateDNSName}}\n  groups:\n    - system:bootstrappers\n    - system:nodes\n- rolearn: ${var.user_role_arn}\n  username: cluster-admin\n  groups:\n    - system:masters\n"
```

Comment:
Is there another way to format this?
It’s just awkward to read and I suspect prone to errors given the indents. How about the <EOF...EOF multiline approach as is used elsewhere?